google/cloud/exceptions.py

```diff
@@ -41,7 +43,10 @@ def __init__(self, message, errors=()):
         self._errors = errors
 
     def __str__(self):
-        return '%d %s' % (self.code, self.message)
+        result = u'%d %s' % (self.code, self.message)
+        if six.PY2:
+            result = _to_bytes(result)
```

Comment:
Before I added that, this path seems to still be broken anyway for py27.
It looks like `_to_bytes` can't handle that unicode character.

```bash
>                 if isinstance(value, six.text_type) else value)
E       UnicodeEncodeError: 'ascii' codec can't encode character u'\u2019' in position 8: ordinal not in range(128)

.tox/py27/lib/python2.7/site-packages/google/cloud/_helpers.py:481: UnicodeEncodeError
```